src/components/elements/table/tableRowActionUtil.tsx

```diff
@@ -30,15 +38,50 @@ export const getViewClientMenuItem = (client: ClientNameFragment) => {
 
 export const getViewEnrollmentMenuItem = (
   enrollment: Pick<EnrollmentFieldsFragment, 'id' | 'entryDate' | 'exitDate'>,
-  client: ClientNameFragment
+  client: Pick<ClientNameFragment, 'id'> | ClientNameFragment
 ) => {
   return {
     title: 'View Enrollment',
     key: 'enrollment',
-    ariaLabel: `View Enrollment, ${clientBriefName(client)} ${entryExitRange(enrollment)}`,
+    ariaLabel: `View Enrollment, ${client.hasOwnProperty('firstName') ? clientBriefName(client as ClientNameFragment) : ''} ${entryExitRange(enrollment)}`,
```

Comment:
Type cast shouldn't be needed  / I don't think `Pick<ClientNameFragment, 'id'> ` should be permitted -- I think this is same as other comment on the household PR?

Reply:
Hm, yes it's the same comment, sorry I didn't mention something; I moved this change over from the other PR, where it wasn't relevant. But I agree with you that it's a bit messy and could be improved. It fixes the typescript issue arising in `ClientServicesPage`, where we don't have a client fragment to pass, only client ID. The Client Services Page overrides the aria label that calls for the client ID anyhow, so it's purely a typescript issue.

A couple alternatives ideas:
- Just avoid `getViewEnrollmentMenuItem` on `ClientServicesPage` - ok, but leads to some duplication
- Use `client` from dashboard context - seems like a bit of a hassle to keep Typescript happy, but maybe ok?

What do you prefer? I think I lean slightly towards the second option.

Reply:
Ah I see, thanks. I didn't realize there really was a called that only passed `id`. I think the approach you added with https://github.com/greenriver/hmis-frontend/pull/998/commits/eee782f11fd4276c955c381161b50d7340c10f3d is fine but we should not need the cast `as ClientNameFragment`